introduction/architecture.md

```diff
@@ -132,20 +132,19 @@ Back-end data nodes store data associated with distributed hypertables. Data nod
 are not aware of cluster-wide distribution policies, and otherwise look just as if they were
 single TimescaleDB instances. However, certain actions are blocked on data nodes to protect
 from accidentally executing operations on data nodes that might endanger the data consistency
-of distributed hypertables.
+of distributed hypertables. There are very few reasons to ever access a data node directly, and
+all accesses should be done at the access node level.
 
 It is important to note that access nodes and data nodes both run TimescaleDB, and for all intents and
-purposes, look just like a single instance of TimescaleDB from an operational perspective.
+purposes, look just like a single instance of TimescaleDB from an operational perspective. However, as
+mentioned above, all accesses should go through the access node, and you should not access the data
+node directly.
 
 ### Distributed Databases
 
-Distributed databases represent the highest level in the hierarchy used to define a multi-node
-TimescaleDB Cluster. Users can define access nodes and data nodes at the database level. This means
-that if you have multiple databases in a TimescaleDB Cluster, you could theoretically have
-different nodes play both the role of access node and data node for different databases.
-We do not typically suggest running in this manner. Instead, we recommend that you
-only have one access node that acts as a query routing layer between your application and
-all of your backend data nodes.
+A multi-node TimescaleDB cluster contains distributed databases. Distributed databases should be configured
```

Comment:
Maybe mention distributed hypertables here to tie everything together: "A multi-node TimescaleDB cluster contains distributed databases on which distributed hypertables can be created."

Reply:
Done!